desk/app/diary.hoon

```diff
@@ -628,21 +630,31 @@
   ::
   ++  di-upgrade
     ^+  di-core
+    ::  we're the host so ignore
```

Comment:
+1 for adding comments to code/logic we want to make sure is exactly right. Rather risk being overly verbose, than not being clear enough.  
That said, could use a little paragraph for `+di-upgrade` as a whole, explaining what "ignoring" and "syncing" imply here.

Gentle recommendations: Official comment style is "flag" comments. And this might be made more legible by being more explicit about the branching.

```suggestion
    ::  if we're the host, no-op
    ::
```